packages/core/src/config/index.js

```diff
@@ -182,8 +175,8 @@ module.exports.init = (userConfig, _logger, defaultsOverride = {}) => {
     targetConfig = Object.assign({}, userConfig);
   }
 
+  // TODO: remove this and forward the logger via init fn.
```

Comment:
This will be the next PR. We don't want to bind the logger to the config object.